python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -259,16 +259,16 @@ def refresh_progress_bar(self, resource_manager: ResourceManager) -> None:
     def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
-        desc = f"- {self.op.name}: {active} active, {queued} queued"
+        desc = f"- {self.op.name}. Tasks: {active} 🟢, {queued} 🟡"
```

Comment:
What do the green and yellow circles represent?

Reply:
green = active, yellow = queued

Reply:
although after reworking the progress bar as suggested above, i have removed the green/yellow emoji